antarest/study/service.py

```diff
@@ -1498,36 +1491,28 @@ def _edit_study_using_command(
             study: study
             url: data path to reach
             data: new data to replace
-            create_missing: Flag to indicate whether to create file or parent directories if missing.
         """
         study_service = self.storage_service.get_storage(study)
         file_study = study_service.get_raw(metadata=study)
         version = file_study.config.version
         commands: List[ICommand] = []
 
         file_relpath = PurePosixPath(url.strip().strip("/"))
-        file_path = study_service.get_study_path(study).joinpath(file_relpath)
-        create_missing &= not file_path.exists()
-        if create_missing:
-            context = self.storage_service.variant_study_service.command_factory.command_context
+        try:
+            tree_node = file_study.tree.get_node(list(file_relpath.parts))
+            commands.append(self._create_edit_study_command(tree_node, url, data, version))
+        except ChildNotFoundError:  # The file doesn't exist, we have to create it.
```

Comment:
It's a little weird that we have a specific "user" command for creation, but not for the edition of an existing one.
Wouldn'it be more consistent to add an additional case in the `_create_edit_study_command` which will create a `CreateUserCommand` resource as soon as `user` starts the path ?
This way, all editions of user resoruces will use that command, instead of having sometimes an `UpdateRawFile` command, depending on the parent state..